### ISSUE TITLE
Don't override default browser font

### ISSUE THREAD

**Issue by Jookia** [Jookia/BGJ2024#11]
The default font is often set for accessibility reasons, we shouldn't override it. Asking Tailwind to stop doing this would be very helpful.

**Comment by Jookia** [comment]
I had some looking in to this and found that this is defined in the normalization/preflight CSS: https://tailwindcss.com/docs/preflight

One thing that kind of rubs me the wrong way is this:

> Unstyled lists are [not announced as lists by VoiceOver](https://unfetteredthoughts.net/2017/09/26/voiceover-and-list-style-type-none/). If your content is truly a list but you would like to keep it unstyled, [add a “list” role](https://www.scottohara.me/blog/2019/01/12/lists-and-safari.html) to the element

I created a PR to fix both these issues.
